main.cpp

```diff
@@ -14,11 +14,15 @@ int main(int argc, char **argv)
 {
     SetupOptions(argc, argv);
 
+    llvm::Expected<std::unique_ptr<REPL>> e_repl = REPL::Create();
+    if(!e_repl)
+        return 1;
+
+    std::unique_ptr<REPL> repl = std::move(*e_repl);
```

Comment:
No need to move, nor any need to make this var. You should be able to just do `e_repl->ExecuteSwift(line)` below.